### ISSUE TITLE
frontend와 backend를 맨 뒤로 옮겨주세요.

### ISSUE THREAD

**Issue by ghyeon0** [team-asdf/yaboja-keyword#3]
키워드 추출 알고리즘 짜기가 그편이 편할 것 같습니다.

{
  "c#": ["c#", "씨샵"],
  "c": ["c언어", "c"],
  "c++": ["c++", "씨플플", "씨쁠쁠"],
  "ruby": ["ruby", "루비"],
  "java": ["java", "자바", "jvm", "jdk"],
  "javascript": ["javascript", "자바스크립트", "jQuery", "제이쿼리", "nodejs", "node.js", "jsp", "spring"],
  "typescript": ["typescript", "타입스크리트"],
  "machinelearning" : ["머신러닝", "ml", "machine", "deep", "reinforcement", "reinforce", "learning", "딥러닝", "기계학습", "강화학습",
             "supervised", "unsupervised", "지도학습", "비지도학습", "학습", "알파고", "gpu", "caffe", "tensorflow", "torch", "theano", "keras", "training",
             "모델", "classification", "분류", "자연어", "빅데이터", "mnist", "손글씨", "비용함수", "gradient", "batch",
             "선형", "회귀", "선형회귀", "linear", "regression", "신경망", "neural", "sigmoid", "relu", "인공지능", "ai"],
  "blockchain" : ["블록체인", "가상화페", "비트코인", "bitcoin", "분산", "분산저장", "p2p", "채굴", "이더리움", "ethereum", "탈중앙"],
  "docker" : ["도커", "docker"],
  "git" : ["깃허브", "깃", "github", "git"],
  "matlab" : ["매트랩", "matlab"],
  "go": ["go", "고"],
  "kotlin": ["kotlin" ,"코틀린"],
  "r": ["r", "알"],
  "swift": ["swift", "스위프트"],
  "database": ["sql", "mysql", "mongodb", "nosql"],
  "php": ["php"],
  "objective-c": ["objective-c"],
  "rust": ["rust", "러스트"],
  "scala": ["scala", "스칼라"],
  "python" : ["python", "python3", "파이썬", "인터프리터", "anaconda", "아나콘다", "pip", "numpy", "scipy", "pandas",
              "jupyter", "scikit", "matplotlib", "django", "장고", "pyqt", "tensorflow", "keras", "flask", "zulip"],
  "frontend": ["웹", "web", "프론트", "프론트엔드", "frontend", "html", "css", "scss", "sass", "stylesheet",
               "스타일시트", "bootstrap", "부트스트랩", "material", "ui", "ux","npm", "webpack", "ajax", "애니메이션",
               "animation", "react", "브라우저", "IE6", "yarn", "stylus", "webpack",
               "리액트", "vue", "angular", "앵귤러", "렌더링", "javascript", "자바스크립트", "typescript", "타입스크립트"],
  "backend" : ["백엔드", "backend", "IO", "멀티쓰레드", "멀티스레드", "api", "tcp", "ip", "aws", "db", "서버", "DBMS",
           "데이터베이스", "mysql", "쿼리", "query", "rdb", "관계형데이터베이스", "sql", "보안", "serverless", "json", "xml",
           "cloud", "클라우드", "트랜잭션"]
}
